bigtable_input.py

```diff
@@ -0,0 +1,135 @@
+"""Read Minigo game examples from a Bigtable.
+"""
+
+import logging
+import os
+import struct
+from google.cloud import bigtable
+from google.cloud.bigtable import row_filters
+import tensorflow as tf
+
+
+_project_name = os.environ['PROJECT']
```

Comment:
can you add "TODO(president.jackson): document these"

Reply:
Done!